src/users/users.service.ts

```diff
@@ -185,6 +185,10 @@ export class UsersService {
 	}
 
 	async deleteUser(userId: string): Promise<void> {
+		await this.resetPasswordRequestsService.deleteResetPasswordRequestForUser(
+			await this.getUser(userId),
+		);
```

Comment:
Couldn't it be possible to add a `onDelete: Cascade` on the Prisma schema as well to stay coherent with the rest of the changes?

Something like:

```text
resetPasswordRequest ResetPasswordRequest? @relation(fields: [resetPasswordRequestId], references: [id], onDelete: Cascade)
```